docs: dev container for GH codespaces (#1070)
sbi-dev/sbi: README.md

## Tutorials

For first time users, you can now head over to the turorials and get going with
[Getting Started](https://sbi-dev.github.io/sbi/tutorial/00_getting_started/).

> The tutorials are easily accessible by opening a codespace on this repo. To do

Author: Added link on GitHub Codespaces in [868b9e0](https://github.com/sbi-dev/sbi/pull/1070/commits/868b9e00bfe7bd1d2efe8bb7a4e447b6f59c26b8).

## Tutorials

> For first time users, you can now head over to the turorials and get going with
> [Getting Started](https://sbi-dev.github.io/sbi/tutorial/00_getting_started/).

Reviewer: ```suggestion
If you're new to `sbi`, we recommend starting with our [Getting
Started](https://sbi-dev.github.io/sbi/tutorial/00_getting_started/) tutorial.
```

> The tutorials are easily accessible by opening a
> [Codespace](https://docs.github.com/en/codespaces/overview) on this repo. To do
> so, click on the green "Code" button and select "Open with Codespaces". This
> provides you with a fully functional environment to run the tutorials as jupyter
> notebooks.

Reviewer: ```suggestion
You can easily access and run these tutorials by opening a
[Codespace](https://docs.github.com/en/codespaces/overview) on this repo. To do
so, click on the green "Code" button and select "Open with Codespaces". This will
provide you with a fully functional environment where you can run the tutorials as
Jupyter notebooks.
```
